pom.xml

```diff
@@ -69,6 +77,16 @@
         <version>${project.version}</version>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-wb-scenario-simulation-editor-api</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-wb-scenario-simulation-editor-backend</artifactId>
```

Comment:
same here: this creates a circular dependency

Reply:
Removed, it is not necessary here